integrations/ros.rst

```diff
@@ -21,14 +21,34 @@ It provides a clean integration that requires no Conan-specific modifications in
 
     **Pre-requisites to run the example:**
 
-    1. In order to run the example, it is exepected that you have an Ubuntu environment (22.04 LTS preferred) with `ROS2 Humble version installed <https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debs.html>`_. You can use the Docker image ``osrf/ros:humble-desktop`` instead as well.
+    1. In order to run the example, it is expected that you have an Ubuntu environment (22.04 LTS preferred) with `ROS2 Humble version installed <https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debs.html>`_. For convenience, you can also use this Docker File instead:
+
+       .. code-block:: docker
+
+           FROM osrf/ros:humble-desktop
+           RUN apt-get update && apt-get install -y \
+           curl \
+           python3-pip \
+           git \
```

Comment:
Add nav2_msgs to support blogpost example at https://github.com/conan-io/examples2/pull/168
```suggestion
           git \
           ros-humble-nav2-msgs \
```